src/test/java/com/softserve/edu/greencity/ui/pages/tipstricks/TipsTricksPage.java

```diff
@@ -170,27 +174,28 @@ public boolean isDesplayedAmountPeople() {
     // amount Bags
 
     public WebElement getAmountBags() {
-        return amountBags;
+        return amountBags = driver.findElement(By.xpath("//app-stat-row/div/div[2]/div/h3"));
```

Comment:
need to rewrite this selector

Reply:
Change to:
amountBags = driver.findElement(By.cssSelector(".stat-row-content:nth-child(2) h3"));